assets/home/sections/extensions.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Col, Grid } from '../grid';
+import Card from '../card';
+import Section from '../section';
+import { useSelect } from '@wordpress/data';
+import { EXTENSIONS_STORE } from '../../extensions/store';
```

Comment:
We'll probably just have one single store for home data, right?

Reply:
That's a good question!

I'm not 100% sure if we'll *need* to have a store for Home data, given that most of the data on Home is pretty static anyway, and we'll have only one single endpoint for it.

Thanks to that, I'm still using only the extensions store, which I didn't move to try to avoid breaking more stuff (given that the extensions store is also used by the editor wizard).

Does it make sense?